fault/magma_simulator_target.py

```diff
@@ -33,13 +47,17 @@ def run(self):
                     value = value.as_uint()
                 simulator.set_value(action.port, value)
             elif isinstance(action, actions.Print):
-                got = BitVector(simulator.get_value(action.port))
+                got = simulator.get_value(action.port)
+                if isinstance(action.port, (m.ArrayType, m.ArrayKind)) and \
+                        isinstance(action.port.T, (m._BitType, m._BitKind)):
+                    got = BitVector(got).as_uint()
+                else if isinstance(action.port, (m.ArrayType, m.ArrayKind)):
+                    raise NotImplementedError("Printing complex nested arrays")
```

Comment:
I think we should defer this feature for now. The user can print elements of the array fine, but there's not a simple way to map a complex nested array to a standard format string.